dlp/snippets/inspect/inspect_test.go

```diff
@@ -500,3 +513,24 @@ func TestInspectImageFile(t *testing.T) {
 		t.Errorf("TestInspectImageFile got %q, want %q", got, want)
 	}
 }
+
+func TestInspectImageFileAllInfoTypes(t *testing.T) {
+	tc := testutil.SystemTest(t)
+	inputPath := "testdata/image.jpg"
+
+	var buf bytes.Buffer
+	if err := InspectImageFileAllInfoTypes(&buf, tc.ProjectID, inputPath); err != nil {
+		t.Errorf("InspectBigQueryTableWithSampling: %v", err)
```

Comment:
Please update the test name prefix to **InspectImageFileAllInfoTypes** in this method instead of _InspectBigQueryTableWithSampling_

Reply:
fixed it.